components/skus/browser/BUILD.gn

```diff
@@ -2,20 +2,58 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//brave/build/cargo.gni")
+
 import("//build/config/features.gni")
 
+lib_build_path = "$root_build_dir/brave_rust/out/$current_cpu"
+lib_output_path =
+    "$lib_build_path/${cargo_lib_prefix}brave_rust${cargo_lib_suffix}"
+
 static_library("browser") {
+  libs = [ lib_output_path ]
+
   sources = [
+    "brave-rewards-cxx/src/cxx.h",
```

Comment:
aren't these generated files?

Reply:
also we really need to start using real paths for these things instead of putting them at the top level

Reply:
You've got this resolved! 👍